consensus/src/chained_bft/consensus_types/timeout_msg.rs

```diff
@@ -49,6 +50,17 @@ impl CryptoHash for PacemakerTimeoutSerializer {
     }
 }
 
+/// TimeoutMsg verification errors.
+#[derive(Debug, Fail, PartialEq)]
+pub enum TimeoutVerificationError {
+    /// The vote message carried by the timeout doesn't pass verification.
+    #[fail(display = "VoteMsgError")]
```

Comment:
Display for this one should contain info from inner element too? E.g. `"{}", _0)` part